utils/sql/src/main/java/io/apicurio/registry/storage/impl/sql/CommonSqlStatements.java

```diff
@@ -147,17 +147,20 @@ public String updateArtifactLatest() {
     public String updateArtifactLatestGlobalId() {
         return "UPDATE artifacts SET latest = (SELECT v.globalId FROM versions v WHERE v.artifactId = ? AND v.version = ?) WHERE artifactId = ?";
     }
-    
+
     /**
      * @see io.apicurio.registry.storage.impl.sql.SqlStatements#insertVersion()
      */
     @Override
-    public String insertVersion(boolean firstVersion) {
+    public String insertVersion(boolean firstVersion, String globalId) {
+        String query;
         if (firstVersion) {
-            return "INSERT INTO versions (artifactId, version, state, name, description, createdBy, createdOn, labels, properties, contentId) VALUES (?, 1, ?, ?, ?, ?, ?, ?, ?, ?)";
+            query = "INSERT INTO versions (globalId, artifactId, version, state, name, description, createdBy, createdOn, labels, properties, contentId) VALUES (%s, ?, 1, ?, ?, ?, ?, ?, ?, ?, ?)";
```

Comment:
Why don't just bind the globalId, like we're doing with the other parameters?

Reply:
I tried that before, and in the case of the sql sequence case triying to bind `nextval('globalidsequence')` does not work.
The library we are using tries to bind that as the value to insert, and in that case it is an expression for the db to generate the value.
And I needed to combine both, allowing the db to generate the value or providing the value beforehand

Reply:
I understand, but I think doing it this way is going to trigger security scanners to flag this as a SQL injection vulnerability (technically it is).  We could bind the value just like the other parameters if we make a separate DB call to get the next sequence value.  What do you think about that approach?

Reply:
if making a separate DB call is fine with transactionality, which I guess it is , I'm ok with your approach

Reply:
Yeah should be fine with the Tx.  Let's try doing that - it should clean some things up.